tests/test_transformers/test_capper.py

```diff
@@ -0,0 +1,15 @@
+from sklego.preprocessing import ColumnsCapper
```

Comment:
there should also be a `test_basics.py` file in the folder where we test estimators. 

since this estimator also has properties we test there it might be nice to add tests there too for the `ColumnsCapper`.